browser-test/src/application_navigation.test.ts

```diff
@@ -0,0 +1,88 @@
+      applicantQuestions = new ApplicantQuestions(pageObject);
+
+      await adminQuestions.addDateQuestion({questionName: 'nav-date-q'});
+      await adminQuestions.addEmailQuestion({questionName: 'nav-email-q'});
+      await adminQuestions.addAddressQuestion({questionName: 'nav-address-q'});
+      await adminQuestions.addRadioButtonQuestion({questionName: 'nav-radio-q', options: ['one', 'two', 'three']});
+      await adminQuestions.addStaticQuestion({questionName: 'nav-static-q'});
+
+      await adminPrograms.addProgram(programName);
+      await adminPrograms.editProgramBlock(programName, 'first description', ['nav-date-q', 'nav-email-q']);
+      await adminPrograms.addProgramBlock(programName, 'second description', ['nav-static-q']);
+      await adminPrograms.addProgramBlock(programName, 'third description', ['nav-address-q']);
+      await adminPrograms.addProgramBlock(programName, 'fourth description', ['nav-radio-q']);
+
+      await adminPrograms.gotoAdminProgramsPage();
+      await adminPrograms.publishProgram(programName);
+
+      await logout(pageObject);
+    });
+
+    it('clicking previous on first block goes to summary page', async () => {
+      await loginAsGuest(pageObject);
+      await selectApplicantLanguage(pageObject, 'English');
+
+      await applicantQuestions.applyProgram(programName);
+      await applicantQuestions.clickPrevious();
+
+      // Assert that we're on the preview page.
+      expect(await pageObject.innerText('h1')).toContain('Program application review');
+    });
+
+    it('clicking previous on later blocks goes to previous blocks', async () => {
+      await loginAsGuest(pageObject);
+      await selectApplicantLanguage(pageObject, 'English');
+
+      await applicantQuestions.applyProgram(programName);
+
+      // Fill out the first block and click next
+      await applicantQuestions.answerDateQuestion('2021-11-01');
+      await applicantQuestions.answerEmailQuestion('test1@gmail.com');
+      await applicantQuestions.clickNext();
+
+      // Nothing to fill in since this is the static question block
+      await applicantQuestions.clickNext();
+
+      // Fill out address question and click next
+      await applicantQuestions.answerAddressQuestion('1234 St', 'Unit B', 'Sim', 'Ames', '54321');
+      await applicantQuestions.clickNext();
+
+      // Click previous and see file upload page with address 
+      await applicantQuestions.clickPrevious();
+      await applicantQuestions.checkAddressQuestionValue('1234 St', 'Unit B', 'Sim', 'Ames', '54321');
+
+      // Click previous and see static question page
+      await applicantQuestions.clickPrevious();
+      await applicantQuestions.seeStaticQuestion('static question text');
+
+      // Click previous and see date and name questions
+      await applicantQuestions.clickPrevious();
+      await applicantQuestions.checkDateQuestionValue('2021-11-01');
+      await applicantQuestions.checkEmailQuestionValue('test1@gmail.com');
+
```

Comment:
Is there a point where we don't see the "Previous" button anymore? If so, we should add a check to expect that we don't see it (whenever that is, i'd assume on the first block).

Reply:
Or...can we check that the final "Previous" button brings you back to the initial Preview page?

Reply:
The first test in this file actually tests that the first previous brings you to the review page. Added here too though.